src/wr_logic_ai/src/drive_mux_select.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+import rospy
+from wr_logic_ai.msg import NavigationState
+from std_msgs.msg import String
+from typing import Dict
+from topic_tools.srv import MuxSelect
+
+current = -1
+class MUX:
+    def __init__(self) -> None:
+        self.last_message = None
+
+    def state_call_back(self, message: NavigationState, conversion_table: Dict[int, str], mux_name : str):
+        if (self.last_message != message):
+            self.last_message = message
```

Comment:
When comparing/storing the message, use the internal variable name directly, rather than the message as a whole, for stability.
```suggestion
        if (self.last_message != message.state):
            self.last_message = message.state
```